shell/platform/windows/windows_proc_table.cc

```diff
@@ -17,7 +20,7 @@ WindowsProcTable::~WindowsProcTable() {
 }
 
 BOOL WindowsProcTable::GetPointerType(UINT32 pointer_id,
-                                      POINTER_INPUT_TYPE* pointer_type) {
+                                      POINTER_INPUT_TYPE* pointer_type) const {
```

Comment:
Is making these `const` part of the move this PR introduces, or is it just a separate minor update?

Reply:
It's a separate minor update. These should have been `const` all along